docs/DeveloperGuide.md

```diff
@@ -1040,6 +1040,34 @@ testers are expected to do more *exploratory* testing.
 
 3. _{ more test cases … }_
 
+### Rooms
+
+1. Adding a room
+
+    1. Prerequisites: There are no rooms in SunRez, or the room with the number `10-999` does not exist in the system.
+
+    2. Test case: `oadd r/10-999 t/suite_ac` <br>
+       Expected: A feedback message in the result box indicating the room was successfully added. The room should also be visible in the room panel.
+
+    3. Test case: `oadd r/10-999 t/suite_ac` <br>
+       Expected: An error message indicating the room `10-999` already exists in the system.
+
+    4. Test case: `oadd r/00-999 t/suite_ac` <br>
+        Expected: An error message indicating the value constraints for room number
+
+1. Deleting a room
+
+    1. Prerequisites: The room with the number `10-999` must exist in the system and its index must be known.
+
+    2. Test case: `odel [index of room 10-999]` <br>
+       Expected: A feedback message in the result box indicating the room was successfully deleted. The room should no longer be visible in the room panel.
+
+    3. Test case: `odel -5` <br>
+       Expected: An error message indicating that the index must be a non-zero unsigned integer.
```

Comment:
This might need a little rephrasing as we decided on a slightly different error message

Reply:
Updated.